src/PowerShellEditorServices.Protocol/MessageProtocol/MessageReader.cs

```diff
@@ -110,20 +110,32 @@ public async Task<Message> ReadMessage()
             // Get the JObject for the JSON content
             JObject messageObject = JObject.Parse(messageContent);
 
-            // Load the message
-            this.logger.Write(
-                LogLevel.Diagnostic,
-                string.Format(
-                    "READ MESSAGE:\r\n\r\n{0}",
-                    messageObject.ToString(Formatting.Indented)));
-
-            // Return the parsed message
+            // Deserialize the message from the parsed JSON message
             Message parsedMessage = this.messageSerializer.DeserializeMessage(messageObject);
 
-            this.logger.Write(
-                LogLevel.Verbose,
-                $"Received {parsedMessage.MessageType} '{parsedMessage.Method}'" +
-                (!string.IsNullOrEmpty(parsedMessage.Id) ? $" with id {parsedMessage.Id}" : string.Empty));
+            // Log message info - initial capacity for StringBuilder varies depending on whether
+            // the log level is Diagnostic where JsonRpc message payloads are logged and 
+            // vary from 1K up edited file size chars.  When not logging message payloads the 
```

Comment:
```suggestion
            // vary from 1K up to edited file size chars.  When not logging message payloads the 
```
Is this what you meant?

Reply:
Yup, will fix tonight.